x-pack/plugins/security_solution/public/hosts/components/kpi_hosts/risky_hosts/index.tsx

```diff
@@ -0,0 +1,154 @@
+ * 2.0.
+ */
+
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiHealth,
+  EuiHorizontalRule,
+  EuiIcon,
+  EuiPanel,
+  EuiTitle,
+  EuiText,
+  transparentize,
+} from '@elastic/eui';
+import React from 'react';
+import styled, { css } from 'styled-components';
+import { euiLightVars } from '@kbn/ui-shared-deps-src/theme';
+import { InspectButtonContainer, InspectButton } from '../../../../common/components/inspect';
+
+import { HostsKpiBaseComponentLoader } from '../common';
+import * as i18n from './translations';
+
+import {
+  HostRiskSeverity,
+  HostsKpiRiskyHostsStrategyResponse,
+} from '../../../../../common/search_strategy/security_solution/hosts/kpi/risky_hosts';
+
+import { useInspectQuery } from '../../../../common/hooks/use_inspect_query';
+import { useErrorToast } from '../../../../common/hooks/use_error_toast';
+
+const QUERY_ID = 'hostsKpiRiskyHostsQuery';
+
+const HOST_RISK_SEVERITY_COLOUR = {
+  Unknown: euiLightVars.euiColorMediumShade,
+  Low: euiLightVars.euiColorVis0,
+  Moderate: euiLightVars.euiColorWarning,
+  High: euiLightVars.euiColorVis9_behindText,
+  Critical: euiLightVars.euiColorDanger,
+};
+
+const HostRiskBadge = styled.div<{ $severity: HostRiskSeverity }>`
+  ${({ theme, $severity }) => css`
+    width: fit-content;
+    padding-right: ${theme.eui.paddingSizes.s};
+    padding-left: ${theme.eui.paddingSizes.xs};
+
+    ${($severity === 'Critical' || $severity === 'High') &&
+    css`
+      background-color: ${transparentize(theme.eui.euiColorDanger, 0.2)};
+      border-radius: 999px; // pill shaped
+    `};
+  `}
+`;
+
+const HostRisk: React.FC<{ severity: HostRiskSeverity }> = ({ severity }) => (
+  <HostRiskBadge color={euiLightVars.euiColorDanger} $severity={severity}>
+    <EuiHealth color={HOST_RISK_SEVERITY_COLOUR[severity]}>{severity}</EuiHealth>
+  </HostRiskBadge>
+);
+
+const HostCount = styled(EuiText)`
+  font-weight: bold;
+`;
+HostCount.displayName = 'HostCount';
+
+const StatusTitle = styled(EuiTitle)`
+  text-transform: lowercase;
+`;
+
+const RiskyHostsComponent: React.FC<{
+  error: unknown;
+  loading: boolean;
+  data?: HostsKpiRiskyHostsStrategyResponse;
+}> = ({ error, loading, data }) => {
+  useInspectQuery(QUERY_ID, loading, data);
+  useErrorToast(i18n.ERROR_TITLE, error);
+
+  if (loading) {
+    return <HostsKpiBaseComponentLoader />;
+  }
+
+  const criticalRiskCount = data?.riskyHosts.Critical ?? 0;
+  const hightlRiskCount = data?.riskyHosts.High ?? 0;
+
+  const totalCount = criticalRiskCount + hightlRiskCount;
+
+  return (
+    <InspectButtonContainer>
+      <EuiPanel hasBorder data-test-subj="risky-hosts">
+        <EuiFlexGroup gutterSize={'none'}>
+          <EuiFlexItem>
+            <EuiTitle size="xxxs">
+              <h6>{i18n.RISKY_HOSTS}</h6>
+            </EuiTitle>
+          </EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            {data?.inspect && (
+              <InspectButton queryId={QUERY_ID} title={`KPI ${i18n.RISKY_HOSTS}`} />
```

Comment:
I think it's ok to put `KPI` into `i18n.RISKY_HOSTS` and let the translator to decide how to translate it.